NOpenCL/CommandQueue.cs

```diff
@@ -8,7 +8,10 @@ namespace NOpenCL
     using System;
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
+    using System.Diagnostics;
+    using System.Runtime.InteropServices;
 
+    [DebuggerDisplay("Device = {Device.Name}")]
```

Comment:
This seems like a fine improvement.

* :question: Have you tested this to make sure it behaves properly?
* :question: Are you comfortable sending this as a separate pull request?